OpenRA.Mods.Common/Traits/AppearsOnRadar.cs

```diff
@@ -42,6 +42,14 @@ public void Created(Actor self)
 		{
 			var color = modifier != null ? modifier.RadarColorOverride(self) : self.Owner.Color.RGB;
 
+			if (Game.Settings.Game.UsePlayerStanceColors)
+			{
+				color = self.Owner.PlayerStanceColor(self, modifier);
+
+				if (color == Color.Tan)
```

Comment:
Could we check against the player stance (maybe via an out parameter?) instead of checking against a magic value?